src/Engine/Physics/Trigger.cpp

```diff
@@ -7,17 +7,17 @@
 
 namespace Physics {
     Trigger::Trigger(const btTransform& transform) {
-        trigger = new btCollisionObject();
+        trigger.reset(new btCollisionObject());
         trigger->setWorldTransform(transform);
     }
 
     btCollisionObject* Trigger::GetCollisionObject() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetCollisionObject' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)